apps/trading/components/rewards-container/rewards-container.tsx

```diff
@@ -0,0 +1,324 @@
+              rewardAsset.quantum
+            )} ${rewardAsset.symbol}`}
+            testId="total-rewards"
+          />
+          <div className="flex flex-col gap-4">
+            <CardTable>
+              <tr>
+                <CardTableTH className="flex items-center gap-1">
+                  {t(`Locked ${rewardAsset.symbol}`)}
+                  <VegaIcon name={VegaIconNames.LOCK} size={12} />
+                </CardTableTH>
+                <CardTableTD>
+                  {addDecimalsFormatNumberQuantum(
+                    totalLocked.toString(),
+                    rewardAsset.decimals,
+                    rewardAsset.quantum
+                  )}
+                </CardTableTD>
+              </tr>
+              <tr>
+                <CardTableTH>{t(`Vesting ${rewardAsset.symbol}`)}</CardTableTH>
+                <CardTableTD>
+                  {addDecimalsFormatNumberQuantum(
+                    totalVesting.toString(),
+                    rewardAsset.decimals,
+                    rewardAsset.quantum
+                  )}
+                </CardTableTD>
+              </tr>
+              <tr>
+                <CardTableTH>
+                  {t('Available to withdraw this epoch')}
+                </CardTableTH>
+                <CardTableTD>
+                  {addDecimalsFormatNumberQuantum(
+                    totalVestedRewardsByRewardAsset.toString(),
+                    rewardAsset.decimals,
+                    rewardAsset.quantum
+                  )}
+                </CardTableTD>
+              </tr>
+            </CardTable>
+            {totalVestedRewardsByRewardAsset.isGreaterThan(0) && (
+              <div>
+                <TradingButton
+                  onClick={() =>
+                    setViews(
+                      { type: ViewType.Transfer, assetId },
+                      currentRouteId
+                    )
+                  }
+                  size="small"
+                >
+                  {t('Redeem rewards')}
+                </TradingButton>
+              </div>
+            )}
+          </div>
+        </>
+      ) : (
+        <p className="text-sm text-muted">{t('No rewards')}</p>
+      )}
+    </div>
+  );
+};
+
+export const Vesting = ({
+  pubKey,
+  baseRate,
+}: {
+  pubKey: string | null;
+  baseRate: string;
+}) => {
+  const baseRateFormatted = formatPercentage(Number(baseRate));
+  return (
+    <div className="pt-4">
+      <CardStat value={baseRateFormatted + '%'} />
+      <CardTable>
+        <tr>
+          <CardTableTH>{t('Base rate')}</CardTableTH>
+          <CardTableTD>{baseRateFormatted}%</CardTableTD>
+        </tr>
+        {pubKey && (
+          <>
+            <tr>
+              <CardTableTH>{t('Vesting multiplier')}</CardTableTH>
+              <CardTableTD>0%</CardTableTD>
+            </tr>
+            <tr>
+              <CardTableTH>{t('Available to withdraw next epoch')}</CardTableTH>
+              <CardTableTD>0</CardTableTD>
```

Comment:
question: What if there's something in a `vesting` account. Shouldn't we cover that here?

Reply:
or is it impossible now to have anything in `vesting`?

Reply:
When I picked this one up it was agreed to skip over the vesting card. But it now looks like the required apis are in, so I've updated it

Reply:
@bwallacee  @asiaznik  looks like I was using the wrong value here, so back to draft for this one

Reply:
Please take another look